impeller/entity/contents/content_context.cc

```diff
@@ -150,13 +150,26 @@ ContentContext::ContentContext(std::shared_ptr<Context> context)
   if (!context_ || !context_->IsValid()) {
     return;
   }
+#ifdef FML_OS_ANDROID
```

Comment:
TBD: actual feature selection when we have vulkan support

Reply:
Consider having a define for `IMPELLER_SUPPORT_SSBO` that's just `IMPELLER_ENABLE_METAL || IMPELLER_ENABLE_VULKAN`. Or just use the more verbose one here. You'll need some runtime detection, so an update to the Context object to have some method or struct or something that says whether SSBOs are allowed.

Reply:
Oh I see some of the runtime stuff is started.

Reply:
We need to leave this as runtime support as far as I can tell, once Vulkan is on by default. Avoids literring the code base with ifdefs too

Reply:
Can we have this on on `impeller::Context` instead? You can ask the context for its feature set. That way, only ES will return the legacy stuff.

Reply:
Done